.github/workflows/regression.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: regression testing
+
+# Prevent keeping resources busy when a branch/PR is updated
+# https://docs.github.com/en/actions/using-jobs/using-concurrency
+concurrency:
+  group: ${{ github.repository }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+env:
+  GITHUB_TOKEN: ${{ github.token }}
+  DOCKER_BUILDKIT: 1
+  IMAGE: ghcr.io/mabrains/pdk_regression
```

Comment:
We *can* use the docker image like this but we probably can't depend on a docker image under the `mabrains` name. 

We should move it to either;
 * https:/github.com/hdl
 * Same place as https://github.com/google/skywater-pdk-actions/tree/main/run-drc-for-cell-gds-using-magic

Reply:
Sounds great. We build the PDK images using our repo at: https://github.com/mabrains/build_mabrains_packages/tree/main 
HDL team can fork our repo and build it at their end. Or we could use a different way that would have the tools that we need.

Reply:
Is there a specific need to maintain another image?

There are already image for klayout and magic here:
https://hdl.github.io/containers/ToolsAndImages.html

Alternatively we could also use the conda package from:
https://anaconda.org/litex-hub/klayout
https://anaconda.org/litex-hub/magic

Reply:
@proppy I think that's a good starting point for our collaboration. I could do it if you want. But you asked me to add you to the development cycle. Do you want to do that?

I have given you access to the repo: https://github.com/efabless/globalfoundries-pdk-libs-gf180mcu_fd_pv so you would be able to make those changes.

Waiting for your comment on this.

Reply:
I'm also happy to land this PR as it, as long as we log an issue that we need to reconcile the container image (I'm also more than happy to help there!)

We should probably look at consolidating things with https://github.com/efabless/mpw_precheck/ images since it also need physical verification tool + pdks:

Reply:
https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pv/issues/2

@proppy as requested. Let's target that in a different PR.

Reply:
@proppy and @mithro Completed the migration to use conda environment using make-env git repo.

https://github.com/efabless/globalfoundries-pdk-libs-gf180mcu_fd_pv/commit/19cb6f2ac68e59fd009b713a170bceee44487daa

Please review and resolve if you agree with the changes.